src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -404,240 +401,287 @@ def eq_feed_to_cooling_isobaric(b):
-                * feed_conc_ppm
-                * specific_area_coeffs[b.number_effects.value][11]
+                * self.recovery_vol_phase[0, "Liq"]
+                * specific_area_coeffs[num_effect][8]
+                + capacity * feed_temperature * specific_area_coeffs[num_effect][9]
+                + motive_pressure * specific_area_coeffs[num_effect][10]
+                + motive_pressure * feed_conc_ppm * specific_area_coeffs[num_effect][11]
                 + motive_pressure
-                * b.recovery_vol_phase[0, "Liq"]
-                * specific_area_coeffs[b.number_effects.value][12]
+                * self.recovery_vol_phase[0, "Liq"]
+                * specific_area_coeffs[num_effect][12]
                 + motive_pressure
                 * feed_temperature
-                * specific_area_coeffs[b.number_effects.value][13]
-                + motive_pressure
-                * capacity
-                * specific_area_coeffs[b.number_effects.value][14]
-                + 1 * specific_area_coeffs[b.number_effects.value][15]
-                + motive_pressure**2
-                * specific_area_coeffs[b.number_effects.value][16]
-                + capacity**2 * specific_area_coeffs[b.number_effects.value][17]
-                + feed_temperature**2
-                * specific_area_coeffs[b.number_effects.value][18]
-                + b.recovery_vol_phase[0, "Liq"] ** 2
-                * specific_area_coeffs[b.number_effects.value][19]
-                + feed_conc_ppm**2 * specific_area_coeffs[b.number_effects.value][20]
-            )
-
-        @self.Constraint(doc="Convert specific area to m2/kg/s for CAPEX calculation")
-        def eq_specific_area_kg_s(b):
-            return b.specific_area_per_kg_s == pyunits.convert(
-                b.specific_area_per_m3_day / b.feed_props[0].dens_mass_phase["Liq"],
-                to_units=pyunits.m**2 / pyunits.kg * pyunits.s,
+                * specific_area_coeffs[num_effect][13]
+                + motive_pressure * capacity * specific_area_coeffs[num_effect][14]
+                + 1 * specific_area_coeffs[num_effect][15]
+                + motive_pressure**2 * specific_area_coeffs[num_effect][16]
+                + capacity**2 * specific_area_coeffs[num_effect][17]
+                + feed_temperature**2 * specific_area_coeffs[num_effect][18]
+                + self.recovery_vol_phase[0, "Liq"] ** 2
+                * specific_area_coeffs[num_effect][19]
+                + feed_conc_ppm**2 * specific_area_coeffs[num_effect][20]
             )
 
         heating_steam_mass_flow_rate_coeffs = (
             self._get_heating_steam_mass_flow_rate_coeffs()
         )
 
-        @self.Constraint(doc="heating steam mass flow rate surrogate equation")
-        def eq_heating_steam_mass_flow_rate(b):
+        # Heating steam mass flow rate surrogate equation, as a function of the number of effect
+        def _get_heating_steam_mass_flow_rate(num_effect):
```

Comment:
Typically these "hidden" functions (those prepended with underscore  _) are at the end of the file. And it's my preference also... saves a lot of over-scrolling when I am looking through Constraints :)

Reply:
It makes sense! I moved them to the end of the script.